README.md

```diff
@@ -71,10 +71,12 @@ you need:
 
   * [Install JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
   * [Install Bazel](https://docs.bazel.build/versions/master/install.html).
+  * Clone the copybara sorce locally: 
+      * `git clone git@github.com:google/copybara.git`
```

Comment:
Prefer

`git clone https://github.com/google/copybara.git`

as it doesn't require SSH keys setup.